testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -55,7 +59,7 @@ public void testLength() {
             RealmModel realmModel = session.getContext().getRealm();
             PasswordPolicyManagerProvider policyManager = session.getProvider(PasswordPolicyManagerProvider.class);
 
-            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, "length"));
+            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, LengthPasswordPolicyProviderFactory.ID));
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done